editor/src/clj/editor/app_view.clj

```diff
@@ -2121,23 +2150,26 @@ If you do not specifically require different script states, consider changing th
 (handler/defhandler :open-selected-recent-file :global
   (run [prefs app-view workspace project user-data]
     (let [[resource view-type] user-data]
-      (open-resource app-view prefs workspace project resource {:selected-view-type view-type}))))
+      (open-resource app-view prefs workspace project resource {:selected-view-type view-type
+                                                                :use-custom-editor false}))))
 
 (handler/defhandler :open-recent-file :global
   (active? [prefs workspace evaluation-context]
     (recent-files/exist? prefs workspace evaluation-context))
   (run [prefs app-view workspace project]
     (g/with-auto-evaluation-context evaluation-context
       (doseq [[resource view-type] (recent-files/select prefs workspace evaluation-context)]
-        (open-resource app-view prefs workspace project resource {:selected-view-type view-type})))))
+        (open-resource app-view prefs workspace project resource {:selected-view-type view-type
+                                                                  :use-custom-editor false})))))
 
 (handler/defhandler :reopen-recent-file :global
   (enabled? [prefs workspace evaluation-context app-view]
     (recent-files/exist-closed? prefs workspace app-view evaluation-context))
   (run [prefs app-view workspace project]
     (g/with-auto-evaluation-context evaluation-context
       (let [[resource view-type] (recent-files/last-closed prefs workspace app-view evaluation-context)]
-        (open-resource app-view prefs workspace project resource {:selected-view-type view-type})))))
+        (open-resource app-view prefs workspace project resource {:selected-view-type view-type
+                                                                  :use-custom-editor false})))))
```

Comment:
Opened files only get added to the Recent Files list when they are opened in the Defold editor, so we should always reopen them like that.